### ISSUE TITLE
Feature-Request: Graph to Matrix für festgegebene Matrix

### ISSUE THREAD

**Issue by c64emulator** [pknoe3lh/WordClockGenerator#3]
Leider gibt es momentan bei Deinem Generator keine Möglichkeit, die Matrix-Größe vorzugeben.
Ich fände es aber gut, so was zu haben. Grund: Bei ELV ([www.elv.de](www.elv.de)) gibt es 
- das ID100,  
          - Daten: 204 Bildpunkte in einer Anordnung 12 x 17 mit den Maßen 110 x 100 x 15 mm
          - URL: [https://www.elv.de/id100-info-display-tischanzeige-bausatz.html](https://www.elv.de/id100-info-display-tischanzeige-bausatz.html)
- das ID200,
          - Daten: 255 Bildpunkte in einer Anordnung 15 x 17 mit den Maßen 260 x 260 x 21 mm
          - URL: [https://www.elv.de/elv-info-display-id200-bausatz.html](https://www.elv.de/elv-info-display-id200-bausatz.html) 

Da nützt es mir nix, wenn Dein Generator einen Vorschlag für 68x2 ausspuckt!

**Comment by im-pro-at** [comment]
Prinzipiell wird für jeden Breite die Lösung berechnet die am wenigsten Höhe braucht. 
Wenn du mehr platz hast dann werden die restlichen LEDs einfach nicht verwendet.

**Comment by c64emulator** [comment]
ja, das ist ja das doofe.  Die Usage ist dann völlig zweitranging. Es sollen die benötigten LED gleichmäßig verteilt sein und ALLE nicht benötigten LED sollen mit "Fill Letters" versehen werden.

**Comment by im-pro-at** [comment]
Ja das wäre ein Option!
